models/certificateBaseTemplates.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * name : certificateBaseTemplates.js.
+ * author : Prajwal.
+ * created-date : 16-May-2024.
+ * Description : Schema for Certificate Base Templates.
+ */
+
+module.exports = {
```

Comment:
@Prajwal17Tunerlabs wasn't this earlier there? Also add migration script to create default templates of all kinds.

Reply:
no it was not there, i have added certificateBaseTemplates and certificateTempaltes controllers
and once i recieve svg templates i will add the script to create default templates